3-networks/envs/shared/README.md

```diff
@@ -0,0 +1,50 @@
+# 3-networks/dev
+
+The purpose of this step is to setup private and restricted shared VPCs with default DNS, NAT (optional), Private Service networking, VPC service controls, onprem dedicated interconnect, onprem VPN and baseline firewall rules for environment dev.
```

Comment:
This is not quite accurate for this env / the code it is executing

Reply:
fixed